lib/jsdom/living/nodes/HTMLTextAreaElement-impl.js

```diff
@@ -69,12 +76,14 @@ class HTMLTextAreaElementImpl extends HTMLElementImpl {
   }
 
   set value(val) {
+    const oldValue = this._getValue();
```

Comment:
This still doesn't seem to match the spec, which uses API value (obtained in jsdom via `_getAPIValue()`).

Reply:
Fixed